submit_cgap/scripts/submit_ontology.py

```diff
@@ -0,0 +1,41 @@
+import argparse
+
+from dcicutils.common import APP_FOURFRONT
+from ..submission import submit_any_ingestion
+from ..utils import script_catch_errors
+
+
+EPILOG = __doc__
+
+
+def main(simulated_args_for_testing=None):
+    parser = argparse.ArgumentParser(  # noqa - PyCharm wrongly thinks the formatter_class is invalid
+        description="Submits a gene list",
+        epilog=EPILOG,
+        formatter_class=argparse.RawDescriptionHelpFormatter
+    )
+    parser.add_argument('ontology_filename', help='a file of ontology data')
```

Comment:
I think this can be a option if the user wants to generate the terms locally but this could be a large file, we might consider instead (or additionally) allowing the generation of this file in the ingester.

Reply:
From what would the ontology be generated? Doesn't it have to have input from the user somehow? I think I'm missing something important here.

(For now the intent was to just have some way of doing submitting, so David can test things, but you're right that if merged it would be user-visible. On the other hand, if this gets complicated, I assume we could evolve this once done to implement other use cases.)

Reply:
The `generate_ontology` command just takes an ontology type alone (or multiple) and will generate the file. We could have the command operate this way as well. See https://github.com/4dn-dcic/fourfront/blob/master/src/encoded/commands/generate_ontology.py

Reply:
You misunderstood my concern. I'm sure you're right about the command. But in order to have permission to run the command by invocation from the outside, I assume the lab and award are potentially used for computing the permissions object. Maybe that's already factored in based on who the person is, and nothing is added by saying "I'm acting in this role", though honestly I think it would be better for hospital business rules if you had to (at least in CGAP) say which role you were acting in, since it creates a firewall on what information an individual case should see, and since a person wearing multiple hats should not necessarily be able to see all the things at once that they are authorized to see separately. I guess we could fix that later, though.

Reply:
@aschroed you may have thoughts here as well, or this may be redundant with the other discussion. I _think_ it's @willronchetti I'm waiting on here, but he may be waiting to see your replies to the other stuff above. It's hard to know in cases like this if I'm assembling all info togetther but I wanted to keep this moving along so we can get some resolution.